api/src/main/java/org/openmrs/module/eptsreports/reporting/cohort/definition/CalculationCohortDefinition.java

```diff
@@ -1,17 +1,17 @@
-/*
- * The contents of this file are subject to the OpenMRS Public License
- * Version 1.0 (the "License"); you may not use this file except in
- * compliance with the License. You may obtain a copy of the License at
- * http://license.openmrs.org
+/**
+ * The contents of this file are subject to the OpenMRS Public License Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy of the
+ * License at http://license.openmrs.org ======= /* The contents of this file are subject to the
+ * OpenMRS Public License Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at http://license.openmrs.org
+ * >>>>>>> master
```

Comment:
this looks like merge conflict marks

Reply:
@Guimino this still needs to be addressed

Reply:
fixed